configure.ac

```diff
@@ -7,8 +7,8 @@ define(_CLIENT_VERSION_RC, 0)
 define(_CLIENT_VERSION_IS_RELEASE, false)
 define(_COPYRIGHT_YEAR, 2020)
 define(_COPYRIGHT_HOLDERS,[The %s developers])
```

Comment:
"The %s corps" might be a better signal of the strength of our commitment to ensuring there is enough Bitcoin for everyone.